### PR TITLE
JAMES-2855 Upgrade Apache commons libs

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2055,7 +2055,7 @@
             <dependency>
                 <groupId>commons-configuration</groupId>
                 <artifactId>commons-configuration</artifactId>
-                <version>1.9</version>
+                <version>1.10</version>
```
Comment (Arsnael) [inline pom.xml]: From maven central repo : https://mvnrepository.com/artifact/commons-configuration/commons-configuration
```
Note: This artifact was moved to: 
org.apache.commons » commons-configuration2
```
Should we try to move to it? https://mvnrepository.com/artifact/org.apache.commons/commons-configuration2/2.5

Comment (chibenwa) [reply pom.xml]: > Should we try to move to it? https://mvnrepository.com/artifact/org.apache.commons/commons-configuration2/2.5

The short answer: we should, but it's a **very complex** task. IMO let's take the quick win then - when we have time - do the upgrade to configuration2...

**Comment by trantienduchn** [comment]
```
org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[2b5cfc388b6b893148491a22c680e071f8efa977] 
[2b5cfc388b6b893148491a22c680e071f8efa977] [ERROR] selectingTwiceARegisteredProtocolWithDifferentDestinationShouldReturnDifferentResults  Time elapsed: 0.003 s  <<< ERROR!
[2b5cfc388b6b893148491a22c680e071f8efa977] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
```

considering pick the common collection upgrade from #2586

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
```
org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getPathsShouldReturnUsedUrls  Time elapsed: 0.137 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] Caused by: java.lang.ClassNotFoundException: org.apache.commons.collections.CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getShouldReturnEmptyWhenUrlNotInUse  Time elapsed: 0.007 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getByPathShouldReturnEmptyWhenUrlNotInUse  Time elapsed: 0.006 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] configureShouldThrowWhenNonValidClassesAreProvided  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectingANonRegisteredProtocolShouldFail  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getUrlsShouldBeEmptyIfNoSelectWerePerformed  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectShouldNotReturnDifferentResultsWhenUsedInAConcurrentEnvironment  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getUrlsResultsShouldNotBeDuplicated  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectingTwiceARegisteredProtocolWithSameDestinationShouldReturnTheSameResult  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getByPathShouldReturnRepositoryWhenUrlExists  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getByPathShouldReturnPreviouslyCreatedMatchingMailRepositories  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectingARegisteredProtocolShouldWork  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getPathsResultsShouldNotBeDuplicatedWithTheSameProtocol  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getByPathShouldReturnEmptyWhenNoMailRepositoriesAreMatching  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] configureShouldNotThrowOnEmptyConfiguration  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getShouldReturnRepositoryWhenUrlExists  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getShouldReturnPreviouslyCreatedMailRepository  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getPathsResultsShouldNotBeDuplicatedWithDifferentProtocols  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getPathsShouldBeEmptyIfNoSelectWerePerformed  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getByPathShouldReturnPreviouslyCreatedMatchingMailRepository  Time elapsed: 0.005 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectShouldNotAddUrlWhenProtocolDoNotExist  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] getUrlsShouldReturnUsedUrls  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] selectingTwiceARegisteredProtocolWithDifferentDestinationShouldReturnDifferentResults  Time elapsed: 0.004 s  <<< ERROR!
[c2cfdf26509fa4266211f41dea723fc5db7ff646] java.lang.NoClassDefFoundError: org/apache/commons/collections/CollectionUtils
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 	at org.apache.james.mailrepository.memory.MemoryMailRepositoryStoreTest.setUp(MemoryMailRepositoryStoreTest.java:66)
[c2cfdf26509fa4266211f41dea723fc5db7ff646] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] Running org.apache.james.mailrepository.memory.MailRepositoryStoreConfigurationTest
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.142 s - in org.apache.james.mailrepository.memory.MailRepositoryStoreConfigurationTest
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] Running org.apache.james.dlp.memory.EventSourcingDLPConfigurationStoreTest
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] Tests run: 13, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.237 s - in org.apache.james.dlp.memory.EventSourcingDLPConfigurationStoreTest
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] Results:
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [INFO] 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR] Errors: 
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
[c2cfdf26509fa4266211f41dea723fc5db7ff646] [ERROR]   MemoryMailRepositoryStoreTest.setUp:66 » NoClassDefFound org/apache/commons/co...
```

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
```
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR] Errors: 
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
[af296cb5ea61755033870aa836f825d5f2985697] [ERROR]   DNSJavaServiceTest.setUp:70 » NoClassDefFound org/apache/commons/collections/C...
```

**Comment by chibenwa** [comment]
```
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldBeAbleToLoadEmptyMappingConfiguration:85->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldCreateServersAsDescribedInXmlConfiguration:45->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldThrowOnConflictingPortConfiguration:78->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldThrowOnEmptyFilterName:93->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldThrowOnEmptyServletName:66->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldThrowOnUnavailableFilterName:99->loadConfiguration:39 » NoClassDefFound
[80f9dcdc5a7607c0aa2942285fd7a8f6ad58c6a2] [ERROR]   JettyHttpServerFactoryTest.shouldThrowOnUnavailableServletName:72->loadConfiguration:39 » NoClassDefFound
```

test this please

File: server/container/spring/pom.xml [hunk 2]
```diff
@@ -92,6 +92,10 @@
             <groupId>com.sun.mail</groupId>
             <artifactId>javax.mail</artifactId>
         </dependency>
+        <dependency>
```
Comment (blackheaven) [inline server/container/spring/pom.xml]: Ordering issue

**Pull request closed without merging**